extensions/beancount-mate/src/components/EmptyView.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { List } from "@raycast/api";
+
+const EmptyView: React.FC = () => {
+  return (
+    <List.EmptyView
+      icon={"https://cdn-icons-png.flaticon.com/512/4160/4160766.png"}
```

Comment:
Could we save this in the extension instead and in 128x128 size - remember credits to the author/service if needed